explorer/Explorer.tsx

```diff
@@ -201,7 +201,10 @@ export class Explorer
             )
         }
 
-        if (this.props.isInStandalonePage) this.setCanonicalUrl()
+        if (this.props.isInStandalonePage) {
+            // expose a more useful title to the Google crawler
```

Comment:
this is wrong :/

Reply:
Ugh, sorry, left behind from experimenting with putting the behaviour there. Removed now